docs/python-configuration.md

```diff
@@ -4,6 +4,89 @@ title: Scope Python Agent configuration
 sidebar_label: Configuration
 ---
 
+The behaviour of the Scope Agent can be modified using environment variables or a configuration file `scope.yml` in the source root of the project.
+
+> If these properties are manually configured, they will be `true` only on encountering the string `true` configured on the environment variable. Any other value will be considered as `false`.
```

Comment:
I would remove this. It is confusing and not always the case in python.

Reply:
removed